### DISCUSSION TITLE
Compiles & Runs Fine for FreeBSD -- Add to list!

### DISCUSSION THREAD

**Discussion by thesunexpress** [qtilities/picom-conf#54] [General]
##### Expected Behavior

Fully (?) functional.
##### Current Behavior

Fully (?) functional.
##### Possible Solution

N/A
##### Steps to Reproduce (for bugs)

1.  N/A

##### System Information

* Distribution & Version:  FreeBSD 13.2 / 14.0
* Qt Version:  QT5
* lxqt-build-tools Version:  N/A
* Package version:  N/A

Hello all, 
Not really an issue, but I'd just like to announce that I've had no problems running `picom-conf` on FreeBSD.
Quite surprising actually, as I was expecting to have to do some mangling of the code/cmake config in order to get it to work... but not so! 
Handy bonus:  Surprised to see `picom-conf` appear in XFCE4-Settings Manager too!
I've got both QT5 & QT6 installed, I built `picom-conf` with QT5 -- haven't yet tried with QT6 so far. I'm not sure if it is supposed to work with QT6 just yet anyways.
Can confirm this works with both FreeBSD-13.2-STABLE & FreeBSD-14.0-STABLE, with Qtilities built & installed.
Kudos to the developers.
Screenshot:
![picom-conf](https://github.com/qtilities/picom-conf/assets/87551480/92c01713-d05f-4355-bf6b-c29321c4cc78)

**Comment by redtide** [comment]
> Hello all,
Not really an issue

Hello! If you agree I'll move this to Discussions.

> Kudos to the developers.

Thank you very much for this post and the kind words!
LXQt people did a good job with compton-conf, so it was quite trivial to update it to work with picom.

> I'm not sure if it is supposed to work with QT6

mmm... README says so, I guess it works... :)
